src/DocTests.jl

```diff
@@ -217,7 +217,7 @@ function eval_repl(block, sandbox, meta::Dict, doc::Documents.Document, page)
                 # see https://github.com/JuliaLang/julia/pull/33864
                 ex = REPL.softscope!(ex)
             end
-            c = IOCapture.iocapture(throwerrors=false) do
+            c = IOCapture.iocapture(throwerrors = :interrupt) do
```

Comment:
Unrelated to this PR; but perhaps the interface would be nicer if you passed an exception type
```julia
IOCapture.iocapture(throwerrors = InterruptException) do
    # ...
end
```

Reply:
I think it's worth considering. Could you open an issue at IOCapture?